phpstan.neon

```diff
@@ -0,0 +1,8 @@
+parameters:
+    level: 7
+    paths:
+        - src/
+    ignoreErrors:
+        - '#Array \(array\<array\<string\|null\>\>\) does not accept null.#'
```

Comment:
Re-read this one, please; it's highly _specific_ - it will only ever match `Array (array<array<string>>) does not accept null` and `Array (array<array<null>>) does not accept null`.  Without it in place, I get a problem flagged with line 131 of ClassMethods during an assignment of a `null` value to a sub-array of the `$extractionMethodsCache` property. I tried a number of `@var` annotation permutations on that property, and could not find a value that would cause phpstan to stop flagging it.

Reply:
The correct type for that property is:

`@var (null|string[])[]`

Or

`@var array<string, null|array<string, string>>`